IniFileParser/Model/KeyDataCollection.cs

```diff
@@ -259,6 +259,16 @@ public KeyData GetKeyData(string keyName)
             return null;
         }
 
+        /// <summary>
+        /// Retrieves all the keys in the collection
+        /// </summary>
+        /// <returns>
+        /// A list of all the key names in the collection
+        /// </returns>
+        public List<string> AllKeys() {
```

Comment:
Needed from above in https://github.com/RekkasGit/E3Next/pull/249/files#diff-4cb2d5baea67c3b465f4684b07deced6b82e3b06f81a6f1df32a7bf0d1522b82R376 when we're looping through all keys of an INI Section.

This wasn't public before, so unsure if there was just no use case for it or if we don't want to expose all the keys in the collection